addons/mail/static/src/discuss/core/common/message_patch.js

```diff
@@ -1,4 +1,24 @@
 import { Message } from "@mail/core/common/message";
 import { MessageSeenIndicator } from "@mail/discuss/core/common/message_seen_indicator";
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
 
 Message.components = { ...Message.components, MessageSeenIndicator };
+
+/** @type {Message} */
+const messagePatch = {
+    get repliesIndicatorTitle() {
+        const numReplies = this.props.message.child_ids.length;
+        return numReplies === 1 ? _t("1 Reply") : _t("%s Replies", numReplies);
+    },
+    get showMessageInReply() {
+        return (
+            super.showMessageInReply &&
+            !this.env.repliesMenu?.message?.eq(this.props.message.parent_id)
+        );
+    },
+    get quickActionCount() {
+        return this.env.repliesMenu?.inside ? 2 : super.quickActionCount;
+    },
```

Comment:
Would have been good enough with `1`, so just the "...", but the "reaction" action uses custom component that only works in inline 😬 (and it's already not great to be a full component when it's just for anchor of emoji picker / quick reaction menu).

So ok for now.